res/data/plist_canm.json

```diff
@@ -0,0 +1,47 @@
+{
+    "type": "ANIM",
+    "contents": [
+        {
+            "narc": 20,
+            "member": 0,
```

Comment:
I think it would be nice to reference the NARC and member by name here and have the build step convert them into IDs on the fly, but need to figure out where the data mapping NARCs and members to indices would be available at this point in the build

Reply:
You could move them from the C constants in `include/constants/narc.h` to be a generated multi-lang constant in `consts/`. Then, you'd have access to these in the Python script that builds the NARCs.